### PR TITLE
Week5

### PR DESCRIPTION
**Pull request by raul-2001**

### CODE REVIEW

File: 03-task-manager/starter/app.js [hunk 1]
```diff
@@ -1 +1,35 @@
-console.log('Task Manager App')
+const express = require('express');
+const app = express();
+const tesks = require('./routes/tasks');
```
Comment (zjpetersen) [inline 03-task-manager/starter/app.js]: nit: typo tasks

File: 03-task-manager/starter/quizAnswers.txt [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+Middleware is a function that have access to the request object, response object and next function in the application request-response cycle. The next function is a function in the Express router which, when invoked, executes the middleware succeeding the current middleware. 
+
+    Execute any code.
+    Make changes to the request and the response objects.
+    End the request-response cycle.
+    Call the next middleware in the stack.
+
+2. What does CRUD stand for?
+
+CRUD stands for Ceate, write, update and delete operations.
```
Comment (zjpetersen) [inline 03-task-manager/starter/quizAnswers.txt:12]: I'm sure this was a typo, but the R stands for read not write.

**Review by zjpetersen — APPROVED** [review]
Keep up the good work!

File: 03-task-manager/starter/quizAnswers.txt (hunk 2 above)
Comment (raul-2001) [reply 03-task-manager/starter/quizAnswers.txt:12]: Ooo yes. R stands for read.  Thanks a lot!

File: 03-task-manager/starter/app.js (hunk 1 above)
Comment (raul-2001) [reply 03-task-manager/starter/app.js]: I got it. tesks is going to be tasks :).
I solve it. Thanks a lot.

**Pull request merged**